app/scripts/services/bindingModalUtils.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('BindingModalUtils', function() {
```

Comment:
Going to probably push up `sortApplications` as well, as I need that for `unbindService` to be able to get the name of the associated app.